src/DashboardService.Models.Dto/Models/TaskInfo.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Models;
+
+public record TaskInfo
```

Comment:
you don't have GroupId here but in request you do

Reply:
++